src/platformdirs/unix.py

```diff
@@ -120,6 +136,27 @@ def _first_item_as_path_if_multipath(self, directory: str) -> Path:
         return Path(directory)
 
 
+def get_user_dirs_folder(key: str) -> Optional[str]:
+    """ ""Return directory from user-dirs.dirs config file"""
+    user_dirs_config_path = os.path.join(Unix().user_config_dir, "user-dirs.dirs")
```

Comment:
Historically we did not support this config file, how come now we need to, and how come we don't need it for the other pahs?

Reply:
The documents directory is not part of the XDG Base Dirs spec that this project uses for its other folders. This config file covers well known user directories that doesn't seem to have a formal standard. An example of where this is necessary is when there is a localized documents folder. Glibc also [uses](https://gitlab.gnome.org/GNOME/glib/blob/2.45.8/glib/gutils.c#L1512) this file. https://docs.gtk.org/glib/func.get_user_special_dir.html

Reply:
Do you have documentation on this config file? Should it impact any other of existing dirs?

Reply:
The file itself gives some useful information.

```
# This file is written by xdg-user-dirs-update
# If you want to change or add directories, just edit the line you're
# interested in. All local changes will be retained on the next run.
# Format is XDG_xxx_DIR="$HOME/yyy", where yyy is a shell-escaped
# homedir-relative path, or XDG_xxx_DIR="/yyy", where /yyy is an
# absolute path. No other format is supported.
# 
XDG_DESKTOP_DIR="$HOME/Desktop"
XDG_DOWNLOAD_DIR="$HOME/Downloads"
XDG_TEMPLATES_DIR="$HOME/Templates"
XDG_PUBLICSHARE_DIR="$HOME/Public"
XDG_DOCUMENTS_DIR="$HOME/Documents"
XDG_MUSIC_DIR="$HOME/Music"
XDG_PICTURES_DIR="$HOME/Pictures"
XDG_VIDEOS_DIR="$HOME/Videos"
```

Users can also add custom directories, but this shouldn't affect anything from the XDG Base spec. [Here](https://freedesktop.org/wiki/Software/xdg-user-dirs/) is a page on the main tool that uses this.

Reply:
Can we link to it here 🙏